src/host/settings.cpp

```diff
@@ -376,33 +374,6 @@ void Settings::SetAltF4CloseAllowed(const bool fAllowAltF4Close)
     _fAllowAltF4Close = fAllowAltF4Close;
 }
 
-bool Settings::IsReturnOnNewlineAutomatic() const
-{
-    return _fAutoReturnOnNewline;
-}
-void Settings::SetAutomaticReturnOnNewline(const bool fAutoReturnOnNewline)
-{
-    _fAutoReturnOnNewline = fAutoReturnOnNewline;
-}
-
-bool Settings::IsGridRenderingAllowedWorldwide() const
-{
-    return _fRenderGridWorldwide;
-}
-void Settings::SetGridRenderingAllowedWorldwide(const bool fGridRenderingAllowed)
-{
-    // Only trigger a notification and update the status if something has changed.
-    if (_fRenderGridWorldwide != fGridRenderingAllowed)
-    {
-        _fRenderGridWorldwide = fGridRenderingAllowed;
-
-        if (ServiceLocator::LocateGlobals().pRender != nullptr)
-        {
-            ServiceLocator::LocateGlobals().pRender->TriggerRedrawAll();

```

Comment:
Do we need to retain this logic? We might miss a global repaint if we don't, and grid lines that had been written will not appear on demand.

Reply:
This redraw was also being done in the `SetConsoleOutputModeImpl` method, so this code was never really necessary. See here:

https://github.com/microsoft/terminal/blob/0eff8c06e32bff6c1e8d39e6505ef8a4c6d8ecbf/src/host/getset.cpp#L443-L454

I should also mention that I wrote some little test apps similar to the functional tests, but which render output to visually confirm the modes are working as expected. And when the `ENABLE_LVB_GRID_WORLDWIDE` mode is toggled, the screen was definitely being refreshed, with the grid line disappearing and reappearing.